test/ctr.bats

```diff
@@ -54,12 +55,12 @@ function teardown() {
 	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run sleep 5
+	wait_until_exit "$ctr_id" "1"
+	[ "$status" -eq 0 ]
 	run crictl inspect --output yaml "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
 	[[ "$output" =~ "reason: Error" ]]
-
```

Comment:
This looks orthogonal.  Maybe keep this empty line?

Reply:
agree